src/flag_gems/ops/rsub.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+import torch
+import triton
+
+from ..utils import pointwise_dynamic
+
+
+@pointwise_dynamic(is_tensor=[True, True, False], promotion_methods=[(0, 1, "DEFAULT")])
+@triton.jit
+def rsub_func(x, y, alpha):
+    return y - x * alpha
+
+
+@pointwise_dynamic(
+    is_tensor=[True, False, False], promotion_methods=[(0, 1, "DEFAULT")]
+)
+@triton.jit
+def rsub_func_tensor_scalar(x, y, alpha):
+    return y - x * alpha
+
+
+@pointwise_dynamic(
+    is_tensor=[False, True, False], promotion_methods=[(0, 1, "DEFAULT")]
+)
+@triton.jit
+def rsub_func_scalar_tensor(x, y, alpha):
+    return y - x * alpha
+
+
+def rsub(A, B, *, alpha=1):
```

Comment:
actually aten::rsub calls the sub kernel. now that you reimplemented it, just register it into the library;)

Reply:
"rsub.py" will be removed.